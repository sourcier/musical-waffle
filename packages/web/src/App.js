import React from 'react'
import { BrowserRouter, Route, Switch } from 'react-router-dom'
import { HelmetProvider } from 'react-helmet-async'
import { Global } from '@emotion/core'
import { hot } from 'react-hot-loader/root'

import Layout from './components/ui/Layout'
import Blog from './components/routes/Blog'
import BlogPage from './components/routes/BlogPage'
// import Home from './routes/Home'
import NotFound from './components/routes/NotFound'
import Contact from './components/routes/Contact'
import ThankYou from './components/routes/Contact/ThankYou'
import { treatments } from './libs/theme'

export default hot((props) => (
  <BrowserRouter>
    <HelmetProvider>
      <Global styles={treatments.body} />
      <Layout>
        <Switch>
          {/* <Route path="/" exact component={Home} /> */}
          <Route path="/contact" exact component={Contact} />
          <Route path="/contact/thank-you" exact component={ThankYou} />
          <Route path="/" exact component={Blog} />
          <Route path="/:slug" exact component={BlogPage} />
          <Route component={NotFound} />
        </Switch>
      </Layout>
    </HelmetProvider>
  </BrowserRouter>
))
