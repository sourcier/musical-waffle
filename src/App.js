import React, { Component } from "react";
import { BrowserRouter, Route, Switch } from "react-router-dom";
import { Masthead } from "./components/Masthead";
import { Home } from "./pages/Home";
import { BlogList } from "./pages/BlogList";
import { NotFound } from "./pages/NotFound";
import { BlogPost } from "./pages/BlogPost";

class App extends Component {
  render() {
    return (
      <BrowserRouter>
        <Masthead />
        <Switch>
          <Route exact path={"/"} component={Home} />
          <Route exact path={"/blog"} component={BlogList} />
          <Route exact path={"/blog/:slug"} component={BlogPost} />
          <Route component={NotFound} />
        </Switch>
      </BrowserRouter>
    );
  }
}

export default App;
