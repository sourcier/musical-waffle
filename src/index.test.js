import React from 'react'
import { shallow } from 'enzyme'
import App from './pages/index'

it('renders without crashing', () => {
    const component = shallow(<App restoreSession={jest.fn()} />)
    expect(component).toMatchSnapshot()
})
