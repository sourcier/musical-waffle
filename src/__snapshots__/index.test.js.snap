// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders without crashing 1`] = `
<Fragment>
  <Meta />
  <Hero>
    <h2>
      Digital Solutions
      <br />
      for Your 
      <span>
        Business
      </span>
    </h2>
  </Hero>
</Fragment>
`;
